src/bridge/cockpitpipechannel.c

```diff
@@ -130,6 +165,28 @@ cockpit_pipe_channel_control (CockpitChannel *channel,
           goto out;
         }
 
+      /* ignore size options if this channel is not a pty or we are in prepare() */
+      if (self->pty && self->pipe)
+        {
+          gushort rows, cols;
+
+          if (!cockpit_pipe_channel_read_window_size_options (message, 0, 0, &rows, &cols))
+            {
+              g_warning ("invalid \"window.rows\" or \"window.cols\" option for stream channel");
```

Comment:
Could we include the channel name for a bit easier debugging? That is ```g_warning("%s: invalid ...", self->name);``` and similar in the message below.